src/dune_engine/execution_parameters.mli

```diff
@@ -23,13 +23,30 @@ val hash : t -> int
 
 val to_dyn : t -> Dyn.t
 
+module Action_output_on_success : sig
```

Comment:
It may be useful to add a comment here, similar to the one we print for `--help`.

Reply:
done